pkgs/apps/orca/default.nix

```diff
@@ -1,18 +1,16 @@
 { lib, stdenv, requireFile, autoPatchelfHook, makeWrapper
 , openmpi, openssh
+, xtb, enableXtb ? true
```

Comment:
I think there is no need to make this optional. Just add xtb as a dependency (or is there a scenario where one would need to turn it off?).

Reply:
Probably not. At least I can't imagine any. Is always on now.